abr_control/arms/ur5/config.py

```diff
@@ -35,6 +35,12 @@ def __init__(self, **kwargs):
         super(Config, self).__init__(
             N_JOINTS=6, N_LINKS=7, ROBOT_NAME='ur5', **kwargs)
 
+        # default
+        # self.START_POS =  [0.70010173, 0.79758257, -1.60718906, 0.99171823,
+        #         1.3501941, 0.0]
+
+        # more like jaco start
+        self.START_POS = [0.698, 0.349, -0.698, 0.523, 1.309, 0]
```

Comment:
similar starting position to jaco

Reply:
same as jaco, i think best to set these by task rather than by arm